### PR TITLE
add code for edx.course.enrollment.activated event

### PR DESCRIPTION
**Pull request by mumarkhan999**
**Trello Link:** [trello card link](https://trello.com/c/zHVxuG6e/21-enrollment-events-edxcourseenrollmentactivated)

**Description:** Added code of caliper event edx.course.enrollment.activated and changed its expected json

**Checks before merge:**

- [x] JSON Reviewed
- [x]   Code Reviewed
- [x]   Unit Test Pass
- [x]   Commits squashed

### CODE REVIEW

File: openedx/features/caliper_tracking/transformers/enrollment_transformers.py [hunk 1]
```diff
@@ -5,23 +5,32 @@
...
+            'type': 'Membership'
+        },
+    })
+    caliper_event['actor'].update({
+        'type': 'Person',
+        'name': current_event['username']
+    })
+    caliper_event['referrer']['type'] = 'WebPage'
+    caliper_event['extensions']['extra_fields'].update({
+        'event_sourse': current_event['event_source'],
```
Comment (imhassantariq) [inline openedx/features/caliper_tracking/transformers/enrollment_transformers.py]: change key name to 'event_source'

Comment (mumarkhan999) [reply openedx/features/caliper_tracking/transformers/enrollment_transformers.py]: done.

File: openedx/features/caliper_tracking/transformers/enrollment_transformers.py [hunk 2]
```diff
@@ -5,23 +5,32 @@
 
 def edx_course_enrollment_activated(current_event, caliper_event):
     """
-    When a student enrolls in a course, the server emits an edx.course.enrollment.activated event.
+    When a student enrolls in a course, the server emits an
+    edx.course.enrollment.activated event.
```
Comment (danialmalik) [inline openedx/features/caliper_tracking/transformers/enrollment_transformers.py]: Here doc-string should describe the method, not the event I think.

**Comment by asamolion** [comment]
Unit tests are failing. Please configure your code such that tests pass. @mumarkhan999

**Pull request merged**